homeassistant/components/lyric/sensor.py

```diff
@@ -217,3 +265,37 @@ def native_value(self) -> StateType:
             return cast(StateType, self.entity_description.value(device))
         except TypeError:
             return None
+
+
+class LyricRoomSensor(LyricRoomEntity, SensorEntity):
+    """Define a Honeywell Lyric sensor."""
+
+    coordinator: DataUpdateCoordinator[Lyric]
+    entity_description: LyricRoomSensorEntityDescription
+
+    def __init__(
+        self,
+        coordinator: DataUpdateCoordinator[Lyric],
+        description: LyricRoomSensorEntityDescription,
+        location: LyricLocation,
+        device: LyricDevice,
+        room: LyricRoom,
+    ) -> None:
+        """Initialize."""
+        super().__init__(
+            coordinator,
+            location,
+            device,
+            room,
+            description.key,
+        )
+        self.entity_description = description
+
+    @property
+    def native_value(self) -> StateType:
+        """Return the state."""
+        room: LyricRoom = self.room
+        try:
+            return cast(StateType, self.entity_description.value(room))
```

Comment:
Please avoid casting to state type, the return type of the function should be StateType already